test/unit/io/structure_file/structure_file_input_test.cpp

```diff
@@ -66,7 +66,10 @@ TEST_F(structure_file_input_class, construct_by_filename)
 {
     /* just the filename */
     {
-        test::tmp_filename filename = create_file("> ID\nACGU\n....\n");
```

Comment:
what is create_file is it still used somewhere else?

Reply:
`create_file` is used in a lot of places, I removed it here to test an empty file :point_up: